Code/Tools/AssetProcessor/native/FileWatcher/FileWatcher.cpp

```diff
@@ -25,7 +25,7 @@ static bool IsSubfolder(const QString& folderA, const QString& folderB)
     using AZStd::begin;
     using AZStd::end;
 
-    constexpr auto isSlash = [](const QChar c) constexpr
+    auto isSlash = [](const QChar c) constexpr
```

Comment:
Should we not just remove `isSlash` from the capture groups in the lambdas below? Static storage is automatically captured in lambdas as I understand it.

Reply:
I would also recommend if that doesn't work to at least keep the `const`.

Reply:
That actually makes a lot of sense! Changed

Reply:
Had to reverse, while VS2022 handles it the way you say, VS2019 doesn't like that approach.